docs/cluster-management/configure-api.md

```diff
@@ -160,38 +160,39 @@ datastore:
 
 ### Executor Plugin
 
-We currently support [kubernetes](https://github.com/screwdriver-cd/executor-k8s) and [docker](http://github.com/screwdriver-cd/executor-docker) executor
+We currently support [kubernetes](https://github.com/screwdriver-cd/executor-k8s),  [docker](http://github.com/screwdriver-cd/executor-docker), [VMs in Kubernetes](https://github.com/screwdriver-cd/executor-k8s-vm), and [Jenkins](https://github.com/screwdriver-cd/executor-jenkins) executor. See the [custom-environment-variables file](https://github.com/screwdriver-cd/screwdriver/blob/master/config/custom-environment-variables.yaml) for more details.
 
 #### Kubernetes
 Set these environment variables:
 
 | Environment name   | Default Value | Description                                |
 |:-------------------|:--------------|:-------------------------------------------|
-| EXECUTOR_PLUGIN    |               | Set to `k8s`                               |
-| LAUNCH_VERSION     |               | Launcher version to use                    |
-| K8S_HOST           |               | Kubernetes host                            |
-| K8S_TOKEN          |               | JWT for authenticating Kubernetes requests |
+| EXECUTOR_PLUGIN    | k8s           | Default executor (eg: `k8s`, `docker`, `k8s-vm`, `jenkins`) |
```

Comment:
Petey just added an executor enable field to the API. should mention that
https://github.com/screwdriver-cd/screwdriver/blob/master/config/custom-environment-variables.yaml#L64